### DISCUSSION TITLE
Version and github directory structure

### DISCUSSION THREAD

**Discussion by kutsen** [ncidose/ncidosetools#33] [NCIRF] [answered]
When I enter NCIRF directory, I can see version 2.0.20240301 files there.
![image](https://github.com/user-attachments/assets/cb5921d2-cd92-4fbb-91ca-948c3ef0ef84)
However the link Releases section suggests that the recent version is NCIRF Beta 3.0.202400710 ad for some reason the date is Mar 19. And what does version number 3.0.202400710 in the Releases section mean?
![image](https://github.com/user-attachments/assets/b7105b77-65c8-4113-b045-0321e221f63a)

**Comment by choonsiklee** [comment, marked as answer]
The version 2.0.20240301 is an official version of NCIRF2.0 where only reference size phantoms are included. We're currently running the beta test of NCIRF Beta 3.0.20240710 (sorry 202400710 was a typo and now fixed) where non-reference size phantoms (n=351) are included.
